content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -145,11 +146,11 @@ Run the following commands to set up and deploy Kubeflow.
 
     * **${KF_DIR}** - The full path to your Kubeflow application directory.
 
-1. Check the resources deployed correctly in namespace `kubeflow`
+2. Check the resources deployed correctly in namespace `kubeflow`
 
         kubectl get all -n kubeflow
 
-1. Open the Kubeflow Dashboard
+3. Open the Kubeflow Dashboard
 
     The default installation does not create an external endpoint but you can use port-forwarding to visit your cluster.
     Run the following command and visit 'http://localhost:8080'.
```

Comment:
Perhaps it may sounds better if the step 1 (run the command) is followed by step 2 (open localhost in the browser ).

Note that you should ideally use single back ticks instead of single quotation marks.

```
-    Run the following command and visit 'http://localhost:8080'.
+    Run the following command:

     ```
     kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
     ```

     Next, open `http://localhost:8080` in your browser.
```